examples/tutorials/mvdr_tutorial.py

```diff
@@ -38,258 +51,304 @@
 
-torch.random.manual_seed(0)
-device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+SAMPLE_RATE = 16000
+SAMPLE_MIX = download_asset("tutorial-assets/mvdr/mix.wav")
+SAMPLE_CLEAN = download_asset("tutorial-assets/mvdr/reverb_clean.wav")
 
-print(torch.__version__)
-print(torchaudio.__version__)
-print(device)
 
-filenames = [
-    "mix.wav",
-    "reverb_clean.wav",
-    "clean.wav",
-]
-base_url = "https://download.pytorch.org/torchaudio/tutorial-assets/mvdr"
+######################################################################
+# 2.1. Helper functions
+# ~~~~~~~~~~~~~~~~~~~~~
+#
+
+
+def plot_spectrogram(stft, title="Spectrogram", xlim=None):
+    magnitude = stft.abs()
+    spectrogram = 20 * torch.log10(magnitude + 1e-8).numpy()
+    figure, axis = plt.subplots(1, 1)
+    img = axis.imshow(spectrogram, cmap="nipy_spectral", vmin=-100, vmax=0, origin="lower", aspect="auto")
+    figure.suptitle(title)
+    plt.colorbar(img, ax=axis)
+    plt.show()
+
+
+def plot_mask(mask, title="Mask", xlim=None):
+    mask = mask.numpy()
+    figure, axis = plt.subplots(1, 1)
+    img = axis.imshow(mask, cmap="jet", origin="lower", aspect="auto")
+    figure.suptitle(title)
+    plt.colorbar(img, ax=axis)
+    plt.show()
+
+
+def si_snr(estimate, reference, epsilon=1e-8):
+    estimate = estimate - estimate.mean()
+    reference = reference - reference.mean()
+    reference_pow = reference.pow(2).mean(axis=1, keepdim=True)
+    mix_pow = (estimate * reference).mean(axis=1, keepdim=True)
+    scale = mix_pow / (reference_pow + epsilon)
+
+    reference = scale * reference
+    error = estimate - reference
+
+    reference_pow = reference.pow(2)
+    error_pow = error.pow(2)
+
+    reference_pow = reference_pow.mean(axis=1)
+    error_pow = error_pow.mean(axis=1)
+
+    si_snr = 10 * torch.log10(reference_pow) - 10 * torch.log10(error_pow)
+    return si_snr.item()
 
-for filename in filenames:
-    os.makedirs("_assets", exist_ok=True)
-    if not os.path.exists(filename):
-        with open(f"_assets/{filename}", "wb") as file:
-            file.write(requests.get(f"{base_url}/{filename}").content)
 
 ######################################################################
-# Generate the Ideal Ratio Mask (IRM)
-# -----------------------------------
+# 3. Generate Ideal Ratio Masks (IRMs)
+# ------------------------------------
 #
 
+
 ######################################################################
-# Loading audio data
-# ~~~~~~~~~~~~~~~~~~
+# 3.1. Load audio data
+# ~~~~~~~~~~~~~~~~~~~~
 #
 
-mix, sr = torchaudio.load("_assets/mix.wav")
-reverb_clean, sr2 = torchaudio.load("_assets/reverb_clean.wav")
-clean, sr3 = torchaudio.load("_assets/clean.wav")
-assert sr == sr2
+waveform_mix, sr = torchaudio.load(SAMPLE_MIX)
+waveform_clean, sr2 = torchaudio.load(SAMPLE_CLEAN)
+assert sr == sr2 == SAMPLE_RATE
+# The mixture waveform is a combination of clean and noise waveforms
+waveform_noise = waveform_mix - waveform_clean
```

Comment:
This part felt unintuitive to me. 
If it were `waveform_mix = waveform_clean + waveform_noise` then it makes more sense.
Can we create another asset and fetch it in the same way as the rest?